packages/docusaurus-theme-classic/src/theme/DocCardList/index.tsx

```diff
@@ -18,9 +18,7 @@ export default function DocCardList({
   return (
     <div className="row">
       {items.map((item, index) => (
-        <article key={index} className="col col--6">
```

Comment:
actually passing the col--6 className here was done on purpose:
- The card list is responsible to create the macro layout / grid (ie how many items per row)
- The card item is only responsible to render itself, not deciding how it is rendered inside the macro-layout